docs/hadoop.adoc

```diff
@@ -121,6 +167,187 @@ def defineGratefulDeadSchema(janusGraph) {
+==>tinkerpop.spark activated
+gremlin> graph = GraphFactory.open('conf/hadoop-graph/hadoop-cassandra3.properties')
+==>hadoopgraph[cassandra3inputformat->gryooutputformat]
+gremlin> g = graph.traversal().withComputer(SparkGraphComputer)
+==>graphtraversalsource[hadoopgraph[cassandra3inputformat->gryooutputformat], sparkgraphcomputer]
+gremlin> g.V().count()
+......
+==>808
+gremlin> g.E().count()
+......
+==> 8046
+----
+
+====== 4. Reads from Cassandra backend into Hadoopgraph for OLAP against Spark Cluster in standalone mode.
+
+When running against Standalone Spark cluster, a few extra properties needs to be added, which specifies the CLASSPATH loading order of Spark workers. The updated properties file is similar to the property added in section *Bulk loading into backend data store using Spark Cluster in Standalone mode.*
+
+Use the following properties file to load HadoopGraph against Spark Standalone cluster with Cassandra backend.
+
+[source, properties]
+----
+# spark-cassandra3.properties
+#
+# Hadoop Graph Configuration
+#
+gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
+gremlin.hadoop.graphReader=org.janusgraph.hadoop.formats.cassandra.Cassandra3InputFormat
+gremlin.hadoop.graphWriter=org.apache.tinkerpop.gremlin.hadoop.structure.io.gryo.GryoOutputFormat
+
+gremlin.hadoop.jarsInDistributedCache=true
+gremlin.hadoop.inputLocation=none
+gremlin.hadoop.outputLocation=output
+gremlin.spark.persistContext=true
+
+#
+# JanusGraph Cassandra InputFormat configuration
+#
+janusgraphmr.ioformat.conf.storage.backend=cassandra
+janusgraphmr.ioformat.conf.storage.hostname=127.0.0.1
+janusgraphmr.ioformat.conf.storage.port=9160
+janusgraphmr.ioformat.conf.storage.cassandra.keyspace=janusgraph
+janusgraphmr.ioformat.conf.index.search.backend=elasticsearch
+janusgraphmr.ioformat.conf.index.search.hostname=127.0.0.1
+
+#
+# Apache Cassandra InputFormat configuration
+#
+cassandra.input.partitioner.class=org.apache.cassandra.dht.Murmur3Partitioner
+
+#
+# SparkGraphComputer Configuration
+#
+spark.master=spark://127.0.0.1:7077
+spark.executor.memory=6g
+spark.executor.extraClassPath=/root/lib/janusgraph/*
+spark.serializer=org.apache.spark.serializer.KryoSerializer
+
+----
+
+Then use the properties file as follows from gremlin console:
+
+
+[source, gremlin]
+----
+bin/gremlin.sh
+
+         \,,,/
+         (o o)
+-----oOOo-(3)-oOOo-----
+plugin activated: janusgraph.imports
+gremlin> :plugin use tinkerpop.hadoop
+==>tinkerpop.hadoop activated
+gremlin> :plugin use tinkerpop.spark
+==>tinkerpop.spark activated
+gremlin> graph = GraphFactory.open('conf/hadoop-graph/spark-cassandra3.properties')
+==>hadoopgraph[cassandra3inputformat->gryooutputformat]
+gremlin> g = graph.traversal().withComputer(SparkGraphComputer)
+==>graphtraversalsource[hadoopgraph[cassandra3inputformat->gryooutputformat], sparkgraphcomputer]
+gremlin> g.V().count()
+......
+==>808
+gremlin> g.E().count()
+......
+==> 8046
+----
+
+
 === Running PageRank
```

Comment:
Let's change this to

```=== Other Vertex Programs```

Once you are familiar with how to configure JanusGraph to work with Spark, you can run all the other vertex programs provided by Apache TinkerPop, like Page Rank and Peer Pressure. See the TinkerPop website for more details.